docs/gallery/interactive/fast_fourier_transform.ipynb

```diff
@@ -0,0 +1,381 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "| Normalization| Equation                                                                                                 |\n",
+    "|-------------|----------------------------------------------------------------------------------------------------------|\n",
+    "| `'none'`    | --                                                                                                       |\n",
+    "| `'unitary'` | $X_{\\text{SS}}(k) = \\left\\{ \\begin{array}{ll} X(k) & \\forall k=0, k=\\frac{N}{2} \\\\ 2 X(k) & \\forall 0<k< \\frac{N}{2} \\end{array} \\right.$|\n",
+    "| `'amplitude'` | $\\overline{X}_{\\text{SS}}(k) = \\frac{1}{N} X_{\\text{SS}}(k)$                                          |\n",
+    "| `'rms'`       | $\\overline{X}_{RMS}(k) = \\left\\{ \\begin{array}{ll} \\frac{1}{\\sqrt{2}} \\overline{X}_{\\text{SS}}(k) & \\forall 0<k< \\frac{N}{2} \\\\ \\quad \\overline{X}_{\\text{SS}}(k) & \\forall k=0, k=\\frac{N}{2} \\end{array} \\right.$|\n",
+    "| `'power'`     | $\\overline{\\overline{X}}_{\\text{SS}}(k) = \\lvert \\overline{X}_{\\text{RMS}}(k) \\lvert ^2$           |\n",
+    "| `'psd'`       | $\\overline{\\overline{\\underline{X}}}_{\\text{SS}}(k) = \\frac{N}{f_s} \\overline{\\overline{X}}_{\\text{SS}}(k) = \\frac{N}{f_s} \\lvert \\overline{X}_{\\text{RMS}}(k) \\lvert ^2$|\n",
+    "\n",
+    "Note that the above formulation holds for real-valued signals with single-sided spectra $X_{\\text{SS}}(k)$. Hence, there are small differences in the definitions compared to the formulas written in [Ahrens et al. (2020)](http://www.aes.org/e-lib/browse.cfm?elib=20838)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example signals\n",
+    "Four signals with a length of 100 samples and a sampling rate of 10 kHz are used for illustrating the normalizations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyfar as pf\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "%matplotlib inline\n",
+    "\n",
+    "# set number of samples and sampling rate\n",
+    "n_samples = 1e3\n",
+    "sampling_rate = 10e3"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "An **impulse** ($x(0)=1$ and zero otherwise) with a constant spectrum.\n",
+    "<!-- This is an energy signal, so the appropriate normalization is `'none'` (the default). -->"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "impulse = pf.signals.impulse(n_samples, sampling_rate=sampling_rate)"
```

Comment:
I feel like this could be combined with the previous code block.

Reply:
I'll keep it separated to introduce one signal per code block :)